config/settings_data.json

```
@@ -1 +1,54 @@
-{}
+{
+  "current": "Default",
+  "presets": {
+    "Default": {
+      "sections": {
```

I **think** you need to explicitly repeat the preset values here, otherwise you can't go back to the default preset once you change it.

```suggestion
    "Default": {
      "colors_solid_button_labels": "#FFFFFF",
      "colors_accent_1": "#121212",
      "colors_accent_2": "#334FB4",
      "colors_text": "#121212",
      "colors_outline_button_labels": "#121212",
      "colors_background_1": "#FFFFFF",
      "colors_background_2": "#F3F3F3",
      "type_header_font": "lato_n3",
      "type_body_font": "lato_n3",
      "sections": {
```

To reproduce the issue:
1. Go to `Theme Settings` -> `Theme Style`
2. Change from `Default` to `Craft`. 
3. `Save` and reload the page. You can't go back to `Default` preset.
4. Add the default values like I provided above. Repeat steps 1-3. It should now be possible to go back to default preset.

Maybe I'm doing something wrong or it's a bug with my store, though, so please validate before accepting this suggestion.

---

Ludo and I have both been able to reproduce this and have shared it with the appropriate team. 
That said, if you change the current theme and upload it, it does show the preset correctly with the correct overrides. So I think we can be confident that the issue is not with the presets in the `settings_data` file.

---

Ah but I see what Lucas means. If you set the default to `Craft` in `settings_data.json` which I think would replicate what happens when a merchant uploads a specific style from the theme store or free theme banner in the admin. Then change the style to default... it has nothing to go back to. 
[Video explanation](https://screenshot.click/21-08-3g5lx-1zehp.mp4) (ugh my mic didn't work but basically I set the default as craft and switch back to default and nothing happens)